transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/ListTransferMethodTest.java

```diff
@@ -520,6 +518,11 @@ private String getEndingIn(String ending) {
 
     }
 
+    private String getCardBrandWithFourDigits(String cardBrand,String ending) {
+        return cardBrand+"\u0020\u2022\u2022\u2022\u2022\u0020"+ending;
+
+    }
```

Comment:
1. There should be space between parameters after comma 
2. and there should be space around `+` sign
3. `ending` should be named as `endingDigits`
4. Remove Unnecessary empty line after return

Reply:
fixed